### DISCUSSION TITLE
Global comments off

### DISCUSSION THREAD

**Discussion by kyledenlinger** [knowledgefutures/pubpub#3091] [General]
Working with a professor who is interested in globally disabling comments/discussions in their community. I'm not seeing any way to do that even at the pub level. Is that possible? Is it planned?

**Comment by eastofesten** [comment]
Hi Kyle - we don't have this as a feature at the moment, but I did file a feature request on your behalf. Please feel free to add comments here or to the issue to help us better understand how this could be helpful for your professor's use! 
#3113

**Comment by emmaa-i-ae** [comment]
Hi! I am new to PubPub and also trying to figure out how to disable comments. I am establishing a scholarly journal and it is not appropriate to have commenting on each article (pub).
